update become-the-admin.md (#542)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/misc/become-the-admin.md

4. Check your email for the confirmation code.

> 5. Enter the code to the appropriate field within the form.

Reviewer: ```suggestion
5. Enter the code in the appropriate field in the form.
```
